internal/trace/listener.go

```diff
@@ -60,7 +63,7 @@ func (l *Listener) HandlerStarted(ctx context.Context, msg json.RawMessage) cont
 	if !tracerInitialized {
 		tracer.Start(
 			tracer.WithService("aws.lambda"),
-			tracer.WithLambdaMode(true),
+			tracer.WithLambdaMode(isLambdaMode(extensionPath)),
```

Comment:
I have mix feelings about the naming of this.
Actually we are in lambda mode but in dd-trace-go, lambdaMode is tied to the `logToStdout` boolean so we need to set it to false to use the agent and not to flush traces to logs 
Should we update this method name in dd-trace-go ?

Reply:
Also we might want to remove this logic ? https://github.com/DataDog/dd-trace-go/blob/v1/ddtrace/tracer/option.go#L181 since the go tracer can only be init from here ? it seems to me that this is duplicated logic ?